content/rc/api/examples/manage-subscriptions.md

```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
 }
 ```
 
-To use the sample JSON document in your own account, you must modify these parameters:
+To use the sample JSON document in your own account, modify these parameters:
```

Comment:
I would start this with the action: 
Modify the [following/these] parameters in the sample JSON document to create a subscription on your own account.